src/components/Loading/Loading.tsx

```diff
@@ -8,7 +8,7 @@ import { Wrapper, Badge } from './styles';
 const Loading: React.FC = () => (
   <Wrapper>
     <Badge>
-      <Logo />
+      <Logo md={true} />
```

Comment:
what do you think about a `size` prop taking enums instead which will allow multiple size in future for the Logo component?

Reply:
I'd rather remove the parameter, I think we will never have `90px` there. We even suggest use `40px` in the documentation.

Reply:
Aside of that, it works perfect now.

Reply:
Yeah, that approach was there before #47, so the first step is revert to that moment and later improve with an enum. I've just updated, take a look again @juanpicado @griffithtp